packages/flutter_test/test/finders_test.dart

```diff
@@ -7,6 +7,15 @@ import 'package:flutter/rendering.dart';
 import 'package:flutter_test/flutter_test.dart';
 
 void main() {
+  group('image', () {
+    testWidgets('finds Image widgets', (WidgetTester tester) async {
+      await tester.pumpWidget(_boilerplate(
+        const Image(image: _TestAssetImageProvider('test')),
```

Comment:
Does it mean, if a developer would like to unit test a widget tree that contains `AssetImage`, they would always need to replace the `AssetImage` into something like `_TestAssetImageProvider`? If that's the case, this method is close to unusable.